bin/deploy-vhost

```diff
@@ -723,15 +692,7 @@ sub update_conf_dir {
                 my $psql_user = mySociety::Config::get($psql_conf_prefix.'_DB_USER');
 
                 my $tmp_psql_schema = mktemp("/tmp/deploy-vhost-psql-schema-XXXXXXXX");
-                if ($public) {
-                    chdir_verbose(__LINE__, "$vhost_dir/$vcspath_install");
-                    $git_ref = $conf->{'git_ref'};
-                } else {
-                    chdir_verbose(__LINE__, "$vhost_dir/" . $conf->{'private_git_repository'});
-                    $git_ref = $conf->{'private_git_ref'};
-                }
-                shell("git show $git_ref:$psql_schema >$tmp_psql_schema");
-                chdir_verbose(__LINE__, $mysociety_bin);
+                git("show $git_ref:$psql_schema", { out => $tmp_psql_schema });
                 shell("./psql-schema-compare", $tmp_psql_schema, $conf->{'user'}, $psql_name, $psql_user, $psql_host, $psql_port);
```

Comment:
Don't we need to fully qualify this with `$mysociety_bin`? Even if not, it might be clearer anyway.

Reply:
Yep, well spotted.